src/init/initializeQuiz.js

```diff
@@ -1,33 +1,34 @@
 'use strict';
 
-import { QUESTION_CONTAINER_ID, QUIZ_CONTAINER_ID } from "../constants.js";
-import showCurrentQuestion from "../handlers/showCurrentQuestion.js";
-import createDOMElement from "../utils/createDOMElement.js";
-import getDOMElement from "../utils/getDOMElement.js";
-import createNextQuestionButtonElement from "../views/createNextQuestionButtonElement.js";
+import { QUESTION_CONTAINER_ID, QUIZ_CONTAINER_ID } from '../constants.js';
+import { showCurrentQuestion } from '../handlers/questionHandlers.js';
+import { createDOMElement, getDOMElement } from '../utils/DOMUtils.js';
+import { createNextQuestionButtonElement } from '../views/questionViews.js';
 import { quizData } from '../data.js';
 
 const initializeQuiz = () => {
-    quizData.currentQuestionIndex = 0;
+  quizData.currentQuestionIndex = 0;
 
-    console.log('bla');
-    setupQuizHTML();
+  console.log('bla');
```

Comment:
We can probably remove this now

Reply:
I left it there because it was part of the original. I figured it was there to show `console.log` in action.
I'll remove it.